sqlglot/expressions.py

```diff
@@ -6803,38 +6988,41 @@ def from_(expression: ExpOrStr, dialect: DialectType = None, **opts) -> Select:
 
 def update(
     table: str | Table,
-    properties: dict,
+    properties: t.Optional[dict] = None,
```

Comment:
Why did this become optional?

Reply:
I put it in the PR comments at the top. Making it mandatory prevents you from using the exp.update function when you already have a list SET expressions, so it's not consistent with other helper methods like exp.select

Reply:
Ah, I see; apologies, I didn't notice the description. Will take another look.

Reply:
Sounds reasonable 👍